src/transformers/modeling_encoder_decoder.py

```diff
@@ -191,11 +191,9 @@ def forward(
         input_ids=None,
         inputs_embeds=None,
         attention_mask=None,
-        head_mask=None,
```

Comment:
The values should only optionally be put in via `kwargs` since not every model has a `head_mask` argument (Longformer does not).